### ISSUE TITLE
Can't compile latest release 1.3.6

### ISSUE THREAD

**Issue by Tintin4000** [Azure/azure-iot-arduino#101]
While trying to compile the library I got the following compilation error iothub_client_ll_uploadtoblob.c: 26:27: fatal error: internal/blob.h: No such file or directory
   #include "internal/blob.h".
Seems like the file blob.h is missing.

**Comment by jbobotek** [comment]
Please read the readme to see how to set up. https://github.com/Azure/azure-iot-arduino#simple-sample-instructions

**Comment by Tintin4000** [comment]
Sorry for the previous post, indeed once the instructions were followed no compiling issue anymore.  Thanks.

**Comment by wesgarbee** [comment]
I am having this same issue. I have followed the instructions several times, in several different examples, and I keep getting the same error as OP.

I have added the flags:
`# This can be overriden in boards.txt`
`build.extra_flags=-DESP8266, -DDONT_USE_UPLOADTOBLOB, -DUSE_BALTIMORE_CERT`

I commented out the #define Round():
`// #define round(x)     ((x)>=0?(long)((x)+0.5):(long)((x)-0.5))`

I am using 1.8.11 Arduino IDE and VSCode on Win10. I can provide the entire stack trace if that would help.
